packages/utils/upgrade/src/cli/index.ts

```diff
@@ -15,6 +15,11 @@ const addReleaseUpgradeCommand = (releaseType: Version.ReleaseType, description:
     .option('-n, --dry', 'Simulate the upgrade without updating any files', false)
     .option('-d, --debug', 'Get more logs in debug mode', false)
     .option('-s, --silent', "Don't log anything", false)
+    .option(
+      '-y, --yes',
```

Comment:
We use "force" everywhere else, like data-transfer. The only reason I would switch to "yes" is if we consider force="yes to all", yes="yes to things that are non-destructive" (and thus have both available here)

Reply:
For me, it doesn't mean the same thing. Force means "go for it even if we have weird stuff on the way" whereas "yes" means... answering "yes" to all prompts where you can answer "yes". I don't know if that makes sense though

Reply:
Although for now, I agree it _is_ similar, since the only things we're answering to are optional requirements

Reply:
Yeah, it would technically be 'yes' in this case and then we would introduce 'force' if we encountered something like "run this codemod even if it's unpredictable based on your code" but my worry is that we already use force for data-transfer and now we'd have two different wordings that don't really make that distinction clear. 

@marcoautiero  your thoughts?

Reply:
> but my worry is that we already use force for data-transfer and now we'd have two different wordings that don't really make that distinction clear.

I agree, it was my worry too. But I believe that in the long run having well-documented atomic options/flags, is more helpful than trying to accommodate everything under a big one-do-it-all

Also, `yes` is quite a well-known flag since it's already used quite a lot by package managers & other CLI. So I don't think people would be lost

Reply:
"I had a similar thought when I was going through the code, but for me, it's more of a semantic issue. In our DEITS project, we use 'force' because there are certain actions that go beyond the usual course of the process, such as deleting the destination database, which has a negative connotation. However, in this case, we are agreeing to all the optional requirements, so I don't think that saying yes would cause any irreversible changes to the user project. Please let me know if I'm mistaken."

Reply:
thanks for pinging me; let's keep "yes" for now, we'll evaluate eventual edits while testing it (it's just semantic here, not a big deal)

Reply:
I mean, at least this is my suggestion, I think it's quite aligned to yours

Reply:
We use force to ignore prompts with experimental commands in the strapi CLI by the way. I originally wrote `yes` but Ben highlighted that we use `force` in DTS.

Reply:
Well, we do use force in the DTS, but again, I feel like it's not aiming at doing the same thing :confused: 
I don't really see why both couldn't live together

Reply:
I don't have a problem with yes, but imo if you are ignoring prompts with `yes` i think you need to change the CLI too because that is doing the same thing as here.

Reply:
![Screenshot 2023-12-14 at 16 28 16](https://github.com/strapi/strapi/assets/2686869/f9a96f27-b6fb-46c3-9471-2aa2690ca677)
Do we have potentially destructive effects here?

Reply:
No